WalletWasabi.Fluent/Views/Wallets/Home/History/Columns/LabelsColumnView.axaml

```diff
@@ -8,10 +8,12 @@
              x:Class="WalletWasabi.Fluent.Views.Wallets.Home.History.Columns.LabelsColumnView"
              x:DataType="historyItems:HistoryItemViewModelBase"
              x:CompileBindings="True">
-  <c:PrivacyContentControl PrivacyReplacementMode="Text" NumberOfPrivacyChars="15"
+  <c:PrivacyContentControl PrivacyReplacementMode="Text" NumberOfPrivacyChars="15" FontSize="15"
+                           HorizontalAlignment="Stretch"
+                           VerticalAlignment="Stretch"
                            Classes="monoSpacedPrivacyReplacement">
-    <c:LabelsItemsPresenter Items="{Binding Label}"
-                            HorizontalAlignment="Left"
-                            VerticalAlignment="Center" />
+    <Border BorderBrush="Red" Background="Transparent">
```

Comment:
`BorderBrush` reems redundant here. Also hardcoded color.

Reply:
You're right!